AK/Complex.h

```diff
@@ -224,31 +224,31 @@ class [[gnu::packed]] Complex {
 
 // reverse associativity operators for scalars
```

Comment:
Seems like this comment should be removed?

Reply:
I don't think it should: inside the class there are operators for `Complex+Real`, and here we have "reverse" operators `Real+Complex`.

Reply:
🤷  I read this comment as "`a` and `b` are backwards on purpose below here", but sure.